Declare out when where is used in np.subtract (#9908)
spacetelescope/jwst: changes/9908.residual_fringe.rst

> Fixed residual fringe subtraction that could have caused non-deterministic values under certain conditions.

Reviewer: How about this?
```suggestion
Fixed a bug that introduced arbitrary values to the residual fringe subtraction under certain conditions.
```
